atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -42,16 +42,72 @@ def __init__(self, tasker, workflow, registries):
 
         self.registries = deepcopy(registries)
 
+    def request_delete(self, url, manifest, insecure, auth):
+        response = requests.delete(url, verify=not insecure, auth=auth)
+
+        if response.status_code == requests.codes.ACCEPTED:
+            self.log.info("deleted manifest %s", manifest)
+            return True
+        elif response.status_code == requests.codes.NOT_FOUND:
+            self.log.warning("cannot delete %s: not found", manifest)
+        elif response.status_code == requests.codes.METHOD_NOT_ALLOWED:
+            self.log.warning("cannot delete %s: image deletion disabled on registry",
+                             manifest)
+        else:
+            msg = "failed to delete %s: %s" % (manifest, response.reason)
+            self.log.error("%s\n%s", msg, response.text)
+            raise PluginFailedException(msg)
+
+        return False
+
+    def make_manifest(self, registry, repo, digest):
+        manifest = registry + "/" + repo + "@" + digest
```

Comment:
Nitpick: to avoid unicode/str issues it's often wise to use string formatting for concatenation, e.g.
`manifest = "{registry}/{repo}@{digest}".format(**vars())`